datamap/mv/core/src/main/scala/org/apache/carbondata/mv/datamap/MVDataMapProvider.scala

```diff
@@ -95,8 +95,8 @@ class MVDataMapProvider(
       dataMapSchema.getRelationIdentifier.getTableName,
       true)
     dropTableCommand.processMetadata(sparkSession)
-    DataMapStoreManager.getInstance.unRegisterDataMapCatalog(dataMapSchema)
     DataMapStoreManager.getInstance().dropDataMapSchema(dataMapSchema.getDataMapName)
```

Comment:
please add a comment, why we need this change

Reply:
added